### ISSUE TITLE
[RFE] Support UID/GID changes

### ISSUE THREAD

**Issue by pbrezina** [pbrezina/sssd-test#2248]
Cloned from Pagure issue: https://pagure.io/SSSD/sssd/issue/2244
- Created at 2014-02-13 10:28:32 by [jhrozek](https://pagure.io/user/jhrozek)
- Closed at 2020-03-24 14:11:03 as wontfix
- Assigned to nobody
---

Currently the back ends do not support changing UIDs and GIDs. However, we need to support this feature for the local provider.

### Comments

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2014-02-13 15:51:46](https://pagure.io/SSSD/sssd/issue/2244#226461)**

Fields changed

milestone: NEEDS_TRIAGE => SSSD 1.13 beta
summary: RFE: Support UID/GID changes => [RFE] Support UID/GID changes

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2014-04-10 18:14:30](https://pagure.io/SSSD/sssd/issue/2244#226462)**

Fields changed

rhbz:  => todo

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2014-09-26 18:49:59](https://pagure.io/SSSD/sssd/issue/2244#226463)**

Fields changed

mark:  => 0

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2015-02-10 16:01:53](https://pagure.io/SSSD/sssd/issue/2244#226464)**

Fields changed

milestone: SSSD 1.13 beta => SSSD 1.13 backlog

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2015-02-12 20:13:22](https://pagure.io/SSSD/sssd/issue/2244#226465)**

Mass-moving tickets not planned for the 1.13 release to 1.14

milestone: SSSD 1.13 backlog => SSSD 1.14 beta

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2016-02-16 13:52:05](https://pagure.io/SSSD/sssd/issue/2244#226466)**

This will be required for the local provider.

milestone: SSSD 1.14 beta => SSSD 1.15 beta
sensitive:  => 0

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2017-02-24 14:33:37](https://pagure.io/SSSD/sssd/issue/2244#226467)**

**Metadata Update from @jhrozek**:
- Issue marked as blocked by: #2228
- Issue set to the milestone: SSSD Future releases (no date set yet)

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2020-03-13 13:25:39](https://pagure.io/SSSD/sssd/issue/2244#633137)**

**Metadata Update from @thalman**:
- Custom field blocking reset (from 2228)
- Custom field design_review reset (from 0)
- Custom field mark reset (from 0)
- Custom field patch reset (from 0)
- Custom field review reset (from 0)
- Custom field sensitive reset (from 0)
- Custom field testsupdated reset (from 0)
- Issue close_status updated to: None
- Issue tagged with: Canditate to close

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2020-03-24 14:11:02](https://pagure.io/SSSD/sssd/issue/2244#636614)**

Thank you for taking time to submit this request for SSSD. Unfortunately this issue was not given priority and the team lacks the capacity to work on it at this time.

Given that we are unable to fulfill this request I am closing the issue as wontfix.

If the issue still persist on recent SSSD you can request re-consideration of this decision by reopening this issue. Please provide additional technical details about its importance to you.

Thank you for understanding.

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2020-03-24 14:11:03](https://pagure.io/SSSD/sssd/issue/2244#636615)**

**Metadata Update from @pbrezina**:
- Issue close_status updated to: wontfix
- Issue status updated to: Closed (was: Open)
